src/definitions.js

```diff
@@ -17,4 +17,8 @@ export const SCRAPERS = {
     name: 'Amex',
     loginFields: ['id', 'card6Digits', PASSWORD_FIELD],
   },
+  hapoalim: {
+    name: 'Bank Hapoalim',
+    loginFields: ['id', PASSWORD_FIELD],
```

Comment:
@sebikaplun the website calls the field `קוד משתמש` so I think `id` should be `username`. `id` makes me think Teudat Zehut.